deploy/cperf_ci.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+
+# Usage: ./cperf_ci [BUILD_NUMBER]
+# Necessary tools installed: docker
+#                            docker-compose
+#                            chmod 777 /opt
+
+BUILD_NUMBER=$1
+BASE_DIR="/opt"
+CONFIG_FILENAME="beryllium_RPC_sb_active_stability_mtcbench"
+WORKSPACE=$BASE_DIR"/nstat"
+PARENT_DIRECTORY=$(dirname $WORKSPACE)
+PYTHONPATH=$WORKSPACE
+RESULTS_DIR=$CONFIG_FILENAME"_RESULTS_"$BUILD_NUMBER
+OUTPUT_FILENAME=$CONFIG_FILENAME
+
+# NSTAT cloning on HOST machine
+# ------------------------------------------------------------------------------
+git clone https://github.com/intracom-telecom-sdn/nstat.git $BASE_DIR/nstat -b master
+
+echo "Workspace: $WORKSPACE"
+cd $WORKSPACE/deploy/docker/
+
+# Building NSTAT docker image
+# ------------------------------------------------------------------------------
+sudo docker build -t intracom/nstat .
```

Comment:
this should be omitted, as NSTAT Docker image is supposed to be pulled from Docker Hub